dev/integration_tests/asset_transformation/.gitignore

```diff
@@ -0,0 +1,43 @@
+# Miscellaneous
```

Comment:
Also, since this is such a simple project, consider not adding a full integration_test project and instead making one on-the-fly as other tools integration tests do:

https://github.com/flutter/flutter/blob/master/packages/flutter_tools/test/integration.shard/test_data/basic_project.dart
https://github.com/flutter/flutter/blob/master/packages/flutter_tools/test/integration.shard/test_data/stateless_stateful_project.dart
https://github.com/flutter/flutter/blob/master/packages/flutter_tools/test/integration.shard/test_data/deferred_components_project.dart

Reply:
Hmm, this seems like a decent idea, but perhaps there is value is keeping the project on repo—we could use it as part of the documentation for the feature on flutter.dev. For example we do this for [flavors](https://docs.flutter.dev/deployment/flavors#launching-your-app-flavors):

> For examples of build flavors for [iOS](https://github.com/flutter/flutter/tree/master/dev/integration_tests/flavors/ios), [macOS](https://github.com/flutter/flutter/tree/master/dev/integration_tests/flavors/macos), and [Android](https://github.com/flutter/flutter/tree/master/dev/integration_tests/flavors/android), check out the integration test samples in the [Flutter repo](https://github.com/flutter/flutter/blob/master/dev/integration_tests/flavors/lib/main.dart).

I imagine myself writing a "how to write your own asset-transforming package" section the flutter.dev documentation for the feature, and users might appreciate being able to see a complete example (and not just a snippet). Let me know what you think.

Reply:
Maybe we should add this to https://github.com/flutter/samples.

There is a baseline cost to every flutter project that we keep in the flutter/flutter repo (both pubspec deps and also maintaining the platform-specific directories.

Reply:
If you go with generating this on the fly, I would recommend not using `dart create` to create the dart transformer project, but instead just programmatically creating the files you need, so that upstream changes to the `dart create ...` output won't break the flutter/flutter tree.

Reply:
TIL of the existence of flutter/samples.

I'll 1) proceed with Jenn's suggestion and 2) consider adding a sample to flutter/samples when writing the flutter.dev documentation.

Reply:
Addressed in [create test project on-the-fly instead of having it in-repo](https://github.com/flutter/flutter/pull/145715/commits/d16900fe286531c65918eb7a74544168a8961304)

Reply:
> Hmm, this seems like a decent idea, but perhaps there is value is keeping the project on repo—we could use it as part of the documentation for the feature on flutter.dev. For example we do this for [flavors](https://docs.flutter.dev/deployment/flavors#launching-your-app-flavors):
> 
> > For examples of build flavors for [iOS](https://github.com/flutter/flutter/tree/master/dev/integration_tests/flavors/ios), [macOS](https://github.com/flutter/flutter/tree/master/dev/integration_tests/flavors/macos), and [Android](https://github.com/flutter/flutter/tree/master/dev/integration_tests/flavors/android), check out the integration test samples in the [Flutter repo](https://github.com/flutter/flutter/blob/master/dev/integration_tests/flavors/lib/main.dart).
> 
> I imagine myself writing a "how to write your own asset-transforming package" section the flutter.dev documentation for the feature, and users might appreciate being able to see a complete example (and not just a snippet). Let me know what you think.

Have you reached out to DevRel? They could help with messaging this.  In any case, at the point you need a complete example for documentation, and not integration tests, I agree that samples would be a better spot for it.

Reply:
> Have you reached out to DevRel?

Good idea